backend/repository/data/gateway_config_expected.json

```diff
@@ -882,13 +882,13 @@
 	"RawMiddlewares": {
 		"example": {
 			"name": "example",
-			"schema": "./middlewares/example/example_schema.json",
-			"importPath": "github.com/uber/zanzibar/examples/example-gateway/middlewares/example"
+			"importPath": "github.com/uber/zanzibar/examples/example-gateway/middlewares/example",
+			"schema": "./middlewares/example/example_schema.json"
 		},
 		"example_reader": {
 			"name": "example_reader",
-			"schema": "./middlewares/example_reader/example_reader_schema.json",
-			"importPath": "github.com/uber/zanzibar/examples/example-gateway/middlewares/example_reader"
+			"importPath": "github.com/uber/zanzibar/examples/example-gateway/middlewares/example_reader",
```

Comment:
I'm surprised that this is needed, the module system should provide import paths by convention (and it's part of the basic information provided by all module instances)

Reply:
holdover from the old middleware way-of-doing-things,  will take a stab at rm